test/Pass/nop-domination-check.c

```diff
@@ -0,0 +1,33 @@
+// REQUIRES: solver
+
+// RUN: env SOUPER_NO_EXTERNAL_CACHE=1 SOUPER_INFER_NOP=1 SOUPER_SOLVER=%solver %sclang -O2 -c -o - %s
+
```

Comment:
do you want some kind of CHECK or CHECK-NOT directive here?

Reply:
This test will crash (lit will report the crash) if domination check is disabled. I guess we don't need CHECK or CHECK-NOT.

Reply:
Or I can add a CHECK-NOT for crash dump if you want.

Reply:
@regehr Fixed